helix-core/src/test/java/org/apache/helix/integration/task/TestMaxNumberOfAttemptsMasterSwitch.java

```diff
@@ -71,6 +71,9 @@ public void beforeClass() throws Exception {
 
   @AfterClass
   public void afterClass() throws Exception {
+    String testClassName = this.getShortClassName();
```

Comment:
Instead of doing this, I suggest refactoring the afterClass() method in TaskSynchronizedTestBase or ZkTestBase. In this case, you don't have duplicate code. And all the child classes have it automatically.

Reply:
Good point. Moved to TaskSynchronizedTestBase @afterclass